gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sat Jan 09 14:28:41 CET 2016
+#Tue Apr 05 16:44:31 CEST 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-bin.zip
```

Comment:
Please add this file to .gitignore if it's difficult to keep it from changing. It should not be changed unless very intentional.